src/input/SimModSuite.h

```diff
@@ -440,18 +444,18 @@ void setCases(pGModel model, pACase &meshCase, pACase &analysisCase, MeshAttribu
     }
 
     pGEntity face;
-    for (int i = 0; i < numFaces; i++) {
+    for (auto const& f : AnalysisAtt.faceBound) {
         // Get the face
-        face = GM_entityByTag(model, 2, i + 1);
+        face = GM_entityByTag(model, 2, f.first + 1);
 
         // Add the face to the model association. Note that we passed
         // the Attribute Information Node into the Model Association
         // at the time when the Model Association was created. That prepares
         // the creation of the AttributeVoid on the face as soon as the
         // association process is started
-        if (AnalysisAtt.faceBound[i]!=0) {
-           logInfo(PMU_rank()) << "faceBound[" << i + 1 <<"] =" << AnalysisAtt.faceBound[i];
-           int index = LUT[AnalysisAtt.faceBound[i]];
+        if (f.second!=0) {
+           logInfo(PMU_rank()) << "faceBound[" << f.first + 1 <<"] =" << f.second;
+           int index = LUT[f.second];
```

Comment:
I think this lookup table could be removed now that you have the vector pairs.

Reply:
Thanks for your comment! I updated the whole part (also to get rid of the sprintf...)

Reply:
ok looks good!